docs/server-settings.md

```diff
@@ -25,4 +27,5 @@ To store a string, it needs to be surrounded by double-quotes. For example, a le
 | `webauthn`                  | String  | `enabled` | Whether webauthn is "enabled", "disabled" or "required".'
 | `webauthn.serviceName`      | String  | `a12n-server` | The service name that should appear in Webauthn dialogs.
 | `webauthn.expectedOrigin`   | String  | `null` | The "origin" of this server. This must be set for webauthn to work
-| `webauthn.relyingPartyId`   | String  | `null` | The origin of the application performing the login.
\ No newline at end of file
+| `webauthn.relyingPartyId`   | String  | `null` | The origin of the application performing the login.
+|`seed.users`| boolean | `null` | Whether to populate local database with dummy users | `SEED_USERS`
```

Comment:
This one is still left over ! I appreciate the fixes for the other missing env variables though!